_blueprints/developers_guide.adoc

```diff
@@ -0,0 +1,17 @@
+= Developer's Guide
+
+== Adding new packages
+
+ALMighty uses https://glide.sh[Glide] for package management.  To
+install a new package, you can follow these steps.  The `<URL>` should
+be replaced with the package URL pointing to Github or any other
```

Comment:
Sorry, but it is not a **URL** but a **package name**. See `glide get --help` for more info. Also, please give an example like `github.com/user/project`.